internal/cloudapi/v2/server.go

```diff
@@ -591,7 +591,7 @@ func serializeManifest(ctx context.Context, manifestSource *manifest.Manifest, w
 		}
 	}
 
-	ms, err := manifestSource.Serialize(depsolveResults.PackageSpecs, containerSpecs, ostreeCommitSpecs, despolveResults.RepoConfigs)
+	ms, err := manifestSource.Serialize(depsolveResults.PackageSpecs, containerSpecs, ostreeCommitSpecs, depsolveResults.RepoConfigs)
```

Comment:
This typo fix should be squashed into the previous commit where it's introduced.

Reply:
oh yes, srry must've missed that